changes.d/5721.feat.md

```diff
@@ -0,0 +1 @@
+Allow task's run mode settings to be changed dynamically at runtime using `cylc broadcast`.
```

Comment:
> ...the point of this work is to make Cylc ready for Skip mode, and that requires the run mode to be applied on a task-by-task basis. Having said which, simulation mode will continue to be a whole workflow setting, at least by default, so you're probably right.
> 
> I think we should probably add the word "settings" - because these definately are changed on a task by task basis.

_https://github.com/cylc/cylc-flow/pull/5721#discussion_r1510912272_

But this PR only enables changing simulation mode settings, right?
```suggestion
Allow tasks' simulation mode settings to be changed dynamically at runtime using `cylc broadcast`.
```

Reply:
Yes, the next, hopefully smaller step, is to allow the run mode to be broadcasted at runtime.